supabase/functions/_backend/triggers/schedule_app_stats.ts

```
@@ -0,0 +1,302 @@
+
+export const app = new Hono<MiddlewareKeyVariables>()
+
+/**
+ * Query Cloudflare Analytics Engine for active apps by organization
+ */
+async function queryActiveAppsByOrgCF(c: any, orgIds: string[], daysBack: number = 7): Promise<Map<string, ActiveApp[]>> {
+    const endDate = new Date()
+    const startDate = new Date()
+    startDate.setDate(startDate.getDate() - daysBack)
+
+    const query = `
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(style/indent)